obspy/io/mseed/util.py

```diff
@@ -105,27 +105,40 @@ def get_start_and_end_time(file_or_file_object):
 @deprecated("'getTimingAndDataQuality' has been renamed to "
             "'get_timing_and_data_quality'. Use that instead.")
 def getTimingAndDataQuality(*args, **kwargs):
-    return get_timing_and_data_quality(*args, **kwargs)
+    return get_flags(activity_flags=False, io_flags=False, *args, **kwargs)
 
 
-def get_timing_and_data_quality(file_or_file_object):
+@deprecated("'get_timing_and_data_quality' has been renamed to "
+            "'get_flags'. Use that instead.")
+def get_timing_and_data_quality(*args, **kwargs):
+    return get_flags(activity_flags=False, io_flags=False, *args, **kwargs)
+
+
+def get_flags(file_or_file_object, starttime=None,
+              endtime=None, io_flags=True, activity_flags=True,
+              t_quality=False):
     """
-    Counts all data quality flags of the given Mini-SEED file and returns
+    Counts all data quality, I/O and activity flags
+    of the given Mini-SEED file and returns
     statistics about the timing quality if applicable.
 
     :type file_or_file_object: str or file
     :param file_or_file_object: Mini-SEED file name or open file-like object
         containing a Mini-SEED record.
-
+    :type str
```

Comment:
It also already accepts `UTCDateTime` objects and anything the `UTCDateTime` constructor eats. Please document that.